content/actions/using-workflows/caching-dependencies-to-speed-up-workflows.md

```diff
@@ -102,7 +102,7 @@ For more information, see [`actions/cache`](https://github.com/actions/cache).
           ~/.gradle/wrapper
     ```
   - With `v1` of the `cache` action, only a single path is supported and it must be a directory. You cannot cache a single file.
-- `restore-keys`: **Optional** An ordered list of alternative keys to use for finding the cache if no cache hit occurred for `key`.
+- `restore-keys`: **Optional** A multiline string containing alternative restore keys, with each restore key being on a new line. If no cache hit occurred for `key`, these restore keys are used sequentially in the order provided to find and restore a cache.
```

Comment:
Optional: Perhaps s/being/placed?
Also, are "sequentially" + "in the order provided" repeating the the intent?

Reply:
> Optional: Perhaps s/being/placed?

Yep, I like that!

```suggestion
- `restore-keys`: **Optional** A multiline string containing alternative restore keys, with each restore key placed on a new line. If no cache hit occurred for `key`, these restore keys are used sequentially in the order provided to find and restore a cache.
```

>  Also, are "sequentially" + "in the order provided" repeating the the intent?

Not really; "sequentially" meaning that it searches for them one after the other, and "in the order provided" being the order in which they're checked. I'm open to changing it if you can think of a better wording.

Reply:
Fair enough. 🤷 I can read and interpret it both ways, so I'll leave the final decision to change (or not!) up to you.